rubygem/lib/zeus.rb

```diff
@@ -1,6 +1,11 @@
 # encoding: utf-8
 require 'socket'
 
+# load exact json version from Gemfile.lock to avoid conflicts
+gemfile = "#{ENV["BUNDLE_GEMFILE"] || "Gemfile"}.lock"
+if File.exist?(gemfile) && version = File.read(gemfile)[/^    json \((.*)\)/, 1]
```

Comment:
Probably not a big deal for this check, but should we be using `[[:space:]]` instead of a blank space?